lib/views/track/track_view.dart

```diff
@@ -90,11 +90,15 @@ class _TrackViewState extends ConsumerState<TrackView>
     TrackModel trackModel,
     TrackFilesModel file,
   ) async {
-    await ref.read(playerProvider.notifier).loadSelectedTrack(
-          trackModel: trackModel,
-          file: file,
-        );
-    unawaited(context.push(RouteConstants.playerPath));
+    try {
+      await ref.read(playerProvider.notifier).loadSelectedTrack(
+            trackModel: trackModel,
+            file: file,
+          );
+      unawaited(context.push(RouteConstants.playerPath));
+    } catch (e) {
+      print(e);
+    }
```

Comment:
Why are we just swallowing this exception? What kind of exception might we expect?

Reply:
I apologize for adding this block; it was only for testing purposes.